CHANGELOG.md

```diff
@@ -0,0 +1,101 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+-
+
+## [v0.6.0-rc1] - 2018-10-16
+
+### Fixed
+- Reorganized and cleaned up repository
+- Fixed examples throughout
+
+### Added
```

Comment:
There were a lot of additions for the API, I think query support got improved or added, the /stac endpoint got added and there some changes. I think @hgs-truthe01 can better describe the changes.

We added/changed/fixed (not sure) separate validation instructions and added JSON schemas where missing (e.g. EO).

Reply:
I don't know that the API changed all that much since 0.5.2, at least I'm not sure what it is.
Not putting anything in for that now, but we can add to the changelog before the final 0.6.0 release.

Reply:
Oh, we added the /stac/ endpoint, to provide full compatibility with the Catalog spec in dynamic services. That is a pretty major improvement, making the dynamic API work the same as the static version. But I agree we don't have to put it in for now - we just want 0.6.0 to have a really full summary.